examples/compare_models.ipynb

```diff
@@ -24,14 +24,14 @@
     "import joblib  # only joblib works reliably in notebooks\n",
     "import numpy as np\n",
     "\n",
-    "from impy.constants import GeV\n",
-    "from impy.kinematics import CenterOfMass\n",
-    "import impy.models as im"
+    "from chromo.constants import GeV\n",
+    "from chromo.kinematics import CenterOfMass\n",
+    "import chromo.models as im"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": null,
```

Comment:
I think it is good to ship pre-evaluated notebooks, so that people can see the outputs.

Reply:
OK, done